Vagrantfile

```diff
@@ -98,9 +102,13 @@ if File.exist?(env_specific_config_path)
 end
 
 # Display the main current configuration parameters
-@ui.info "Welcome to Kubernetes playground!"
+@ui.info "Welcome to Kubernetes playground, version " + K8S_PLAYGROUND_VERSION
```

Comment:
I'd rephrase this, if we accept my suggestion above. Let's leave this comment open, we can think about this later.

Reply:
@StefanoSalsano I guess we've to remove this as well

Reply:
done !